common/screenshotter.go

```diff
@@ -229,6 +232,8 @@ func (s *screenshotter) screenshot(
 		if err := os.WriteFile(path, buf, 0o644); err != nil {
 			return nil, fmt.Errorf("saving screenshot to %q: %w", path, err)
 		}
+
+		s.span.SetAttributes(attribute.String("path", path))
```

Comment:
Should we call this statement in `Page.Screenshot` instead of a `span` field into `screenshotter`? To see the overall flow and keep it simple. We can add this field back if needed to add more granularity later on.

Reply:
Changed in [533a9fa](https://github.com/grafana/xk6-browser/pull/1100/commits/533a9fadce6358ca6c70d447cd8b977e4223b7a6).